CONTRIBUTING.md

```diff
@@ -243,18 +243,87 @@ If in doubt, you can always ask for guidance in the Pull Request or on
 [IRC in the #node-dev channel](https://webchat.freenode.net?channels=node-dev&uio=d4).
 
 Feel free to post a comment in the Pull Request to ping reviewers if you are
-awaiting an answer on something.
+awaiting an answer on something. If you encounter words or acronyms that
+seem unfamiliar, check out this
+[glossary](https://sites.google.com/a/chromium.org/dev/glossary).
 
+Note that multiple commits often get squashed when they are landed (see the
+notes about [commit squashing](#commit-squashing)).
 
 ### Step 8: Landing
 
-Once your Pull Request has been reviewed and approved by at least one Node.js
-Collaborators (often by saying LGTM, or Looks Good To Me), and as long as
-there is consensus (no objections from a Collaborator), a
-Collaborator can merge the Pull Request . GitHub often shows the Pull Request as
- `Closed` at this point, but don't worry. If you look at the branch you raised
- your Pull Request against (probably `master`), you should see a commit with
- your name on it. Congratulations and thanks for your contribution!
+In order to get landed, a Pull Request needs to be reviewed and
+[approved](#getting-approvals-for-your-pull-request) by
+at least one Node.js Collaborator. If the Pull Request touches
+more than the documentation, then it also need to pass a
```

Comment:
Ah, I see your point, but what I meant was to stress that doc PR needs to pass the tests too (and possibly explain why , or not if that expose too much details). I didn't mean to remind people that there is a way to skip the tests. Because I think the previous section is not very clear about doc PR needs tests too. After all it could be unexpected to new comers that you need to set up the whole compiler tool chain to get some markdown changes pass the review. But they do need to, so a heads up could be nice.

That said, since in practice this could be flexible, I think I can leave this as-is. Thank you for the clarification!